.travis.yml

```diff
@@ -116,6 +115,9 @@ install:
 # dependencies on metal.yaml and setup.py, unless setup.py already imports them.
 # Install conda tools for packaging and uploading
 - conda install -q python=${MYCONDAPY} cppcheck doxygen anaconda-client ${CONDA_DEPENDENCIES}
+# Deactivate build env and reactivate it to refresh compilers.
+- source deactivate && source $HOME/miniconda/bin/activate
+- hash -r
```

Comment:
Would hash -r alone also work? Aslo, the activate script includes `hash -r`, so I guess it is not needed separately (if you decide to keep the deactivate/activate step).

Reply:
Ah I didn't know that about activate. No, I don't think hash itself works. There's some environmental variables that get set when the activate script is run again and which aren't run the first time. Since we install compilers, the paths are changing.

Reply:
OK. I think you can then remove the `hash -r` because it is already done in of the `miniconda/bin/activate` script.

Reply:
oh, sorry. it is already done. I was looking at the wrong diff. please ignore previous comment.